pertpy/tools/_perturbation_space/_perturbation_space.py

```diff
@@ -364,50 +365,57 @@ def label_transfer(
         self,
         adata: AnnData,
         column: str = "perturbation",
+        column_uncertainty_score_key: str = "perturbation_transfer_uncertainty",
         target_val: str = "unknown",
-        n_neighbors: int = 5,
-        use_rep: str = "X_umap",
+        neighbors_key: str = "neighbors",
     ) -> None:
         """Impute missing values in the specified column using KNN imputation in the space defined by `use_rep`.
 
         Args:
             adata: The AnnData object containing single-cell data.
-            column: The column name in AnnData object to perform imputation on.
+            column: The column name in adata.obs to perform imputation on.
+            column_certainty_score: The column name in adata.obs to store the certainty score of the label transfer.
             target_val: The target value to impute.
-            n_neighbors: Number of neighbors to use for imputation.
-            use_rep: The key in `adata.obsm` where the embedding (UMAP, PCA, etc.) is stored.
+            neighbors_key: The key in adata.uns where the neighbors are stored.
```

Comment:
```suggestion
            neighbors_key: The key in adata.obsp where the neighbors are stored.
```

They are stored in obsp right?

Reply:
The keys pointing to the neighbor informations in `obsp` are saved in `adata.uns['neighbors' / key]`. But I guess we don't need the pointers if we already know where to search for them. Let's change to `.obsp`.

Reply:
On second thought, let's keep it at `.uns`. This is in line with how scanpy does it, e.g. [here](https://github.com/scverse/scanpy/blob/a91bb02b31a637caeee77c71dcd9fbce8437cb7d/src/scanpy/tools/_umap.py#L158).